webapp/src/components/EventStatus.js

```diff
@@ -91,6 +93,19 @@ class EventStatus extends Component {
           
       }
       else if (event.status.application_status === "Withdrawn") {
+        if (event.event_type === "JOURNAL"){
+            return {
+                titleClass: "text-success",
```

Comment:
I don't understand this, why would it say "you have submitted your article" when the application status is withdrawn?